src/tools/map/making/MapProperties.java

```diff
@@ -3,63 +3,25 @@
 import java.awt.Color;
```

Comment:
No, they are dependent changes. Notice that in many of the removed methods we access a property of `MapData`. These often represent the only class-external usage of a `MapData`. Removing the unused methods then removes the only thing preventing those data members from being marked as private.

Removing everything unused does include some independent changes. It is included because removing unused code is a very beneficial cleanup and secondly keeping as one PR avoids the likely merge conflicts.